src/commands/generateEditorConfig.ts

```diff
@@ -5,7 +5,15 @@ import { FileType, Uri, window, workspace } from 'vscode'
  * current vscode settings.
  */
 export async function generateEditorConfig(uri: Uri) {
-	const editorConfigUri = Uri.parse(`${uri.toString()}/.editorconfig`)
+	const workspaceUri =
+		workspace.workspaceFolders && workspace.workspaceFolders[0].uri
+	const currentUri = uri || workspaceUri
+	if (!currentUri) {
+		window.showErrorMessage("Workspace doesn't contain any folders.")
```

Comment:
I'm not understanding why it matters if the workspace doesn't contain any folders. What's the issue with that?

Reply:
Under what scenario would you not be under a folder? Like when you just open vscode and haven't opened anything yet?

Reply:
This situation should never happen because according to the [command definition](https://github.com/editorconfig/editorconfig-vscode/blob/08d184d50a74f383367a8272680cde208a17b7ad/package.json#L49) it should only be available when inside a folder.

I simply added an error message to gracefully handle the situation in a freak edge case.

> I'm not understanding why it matters if the workspace doesn't contain any folders. What's the issue with that?

Maybe the error message that I wrote wasn't clear enough about what I explained just now. Do you have a suggestion for making it clearer?

> Under what scenario would you not be under a folder? Like when you just open vscode and haven't opened anything yet?

Yes, that would be exactly the scenario, even though it should never happen in the real world.

Reply:
Is this what is breaking the tests under linux? I wish I could see the pipeline output, but I don't have permission to access it.

Reply:
I switched everything over the GitHub Actions and all is well! Thanks for the PR!